drivers/pinmux/pinmux_rcar.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2021 IoT.bzh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT renesas_rcar_pinmux
```

Comment:
`drivers/pinmux/pinmux_rcar.c` is not an actual driver since it doesn't implement any public API functions. That's OK, since we're going to deprecate the pinmux API. Let's move this file to `soc/arm/renesas_rcar/gen3/`. As such the pinmux functions will be exposed to rcar drivers only. The public pinmux/pinctrl API can be added later.

Reply:
Thanks, I did move the pinmux driver to the soc directory in the second version of this PR.